components/FileUploader.js

```diff
@@ -35,6 +39,11 @@ class FileUploaderButton extends Component {
     }
   }
 
+  handleClick = evt => {
+    console.log('click');
```

Comment:
Cool 🙂 The change looks ready to go, but please feel free to have another reviewer. Thanks!